<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyMenu</class>
 <widget class="QMainWindow" name="StudyMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="BaseWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 245, 230);</string>
   </property>
   <widget class="QLabel" name="goBackButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>goBack.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="graphsOption">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>461</width>
      <height>661</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>option big.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="graphsName">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>441</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 250, 255);
color: rgb(0, 168, 243);
font: 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Построение графиков&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Забудьте о долгих часах,&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;проведённых с карандашом, &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;линейкой и системой&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;координат&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
